### PR TITLE
docs: 📚 add "known browser issues" section

### LINKED ISSUE #418: docs: 📚 add "known browser issues" section
## Description
In order clarify the users expectations, I suggest to add a "Known Browser Issues" Section in the documentation folder in Storybook. 
Therewith we let the users know, which known bugs in which browsers we accepted as "won't fix" as the issue comes from a browser and not from our components.

1. First example for this would be the missing border radius for focus styles in Safari.
2. Placeholder image is blurry in Chrome
3. Spinner is wobbly in Safari

## Where to put the documentation
Storybook

## DoR
- [x] Item has business value
- [x] Item has been estimated by the team
- [x] Item is clear and well-defined
- [x] Item dependencies have been identified

## DoD
- [ ] Documentation has been created/updated (if applicable)
- [ ] Migration Guide has been created/updated (if applicable)
- [ ] Relevant stories (Features, A11y) are created/updated
- [ ] Implementation works successfully on `feature` branch

### PR DESCRIPTION
**Pull request by Vahid1919**
## Description:
Added a section in Storybook docs detailed the current known browser issues. Closes #418 

## Definition of Reviewable:
*PR notes: Irrelevant elements should be removed.*
- [x] Documentation is created/updated
- [x] relevant tickets are linked
- [x] PR is assigned to project board

### CODE REVIEW

File: packages/components/src/docs/General/6-Known Browser Issues.mdx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { Source, Markdown, Meta } from '@storybook/blocks';
+import playwrightVersion from 'playwright-version';
+import pkg from '../../../package.json';
+
```
Comment (mariohamann) [inline packages/components/src/docs/General/6-Known Browser Issues.mdx:4]:
```suggestion
import { Meta } from '@storybook/blocks';

```

File: packages/components/src/docs/General/6-Known Browser Issues.mdx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+<Meta title="Docs/General/Known Browser Issues" />
+
+# Known Browser Issues
+
+We've identified several browser issues that may affect visual consistency while using Solid Components.
+It's important for developers to be aware of these issues when implementing our components.
+
+## Missing Border Radius for Focus Styles in Safari
+
+In Safari, focus styles may lack border radius, unlike other browsers. This
```
Comment (mariohamann) [inline packages/components/src/docs/General/6-Known Browser Issues.mdx:14]: Please provide concrete examples or say, that this affects every component.

File: packages/components/src/docs/General/6-Known Browser Issues.mdx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+In Chrome, our placeholder images may appear blurry at times. This is not a package issue but Chrome-specific behavior.
+
+## Wobbly Spinner in Safari
+
+The 'sd-spinner' component may experience spinner animation issues in Safari, causing
+them to appear wobbly.
+
+Being aware of these browser-specific issues is essential. While we cannot
+directly resolve them, developers can make informed decisions and implement
+any necessary workarounds when using Solid Components.
```
Comment (mariohamann) [inline packages/components/src/docs/General/6-Known Browser Issues.mdx:28]:
```suggestion
```

You can remove this sentence. Especially as some of the stuff could be fixed with ugly worksarounds, bloating our library.

**Review by mariohamann — CHANGES_REQUESTED** [review]

File: packages/components/src/docs/General/6-Known Browser Issues.mdx (hunk 3 above)
Comment (karlbaumhauer) [reply packages/components/src/docs/General/6-Known Browser Issues.mdx:28]: maybe we can combine the part from @Vahid1919 and add the explanation from @mariohamann as reason why we dont want to fix them (or cant fix them).

**Review by karlbaumhauer — APPROVED** [review]
LGTM, besides requested changes from @mariohamann

File: packages/components/src/docs/General/6-Known Browser Issues.mdx (hunk 2 above)
Comment (Vahid1919) [reply packages/components/src/docs/General/6-Known Browser Issues.mdx:12]: Removing the entire section as this problem seems to have been fixed on Safari's end.

File: packages/components/src/docs/General/6-Known Browser Issues.mdx (hunk 3 above)
Comment (Vahid1919) [reply packages/components/src/docs/General/6-Known Browser Issues.mdx]: @karlbaumhauer 
@mariohamann I have rewritten the conclusion to communicate our intention and not encourage ugly workarounds :)

File: packages/components/src/docs/General/6-Known Browser Issues.mdx [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+We've identified several browser issues that may affect visual consistency while using Solid Components.
+It's important for developers to be aware of these issues when implementing our components.
+
+## Blurry Placeholder Images in Chrome
+
+In Chrome, our placeholder images may appear blurry at times. This is not a package issue but Chrome-specific behavior.
+
+## Wobbly Spinner in Safari
+
+The 'sd-spinner' component may experience spinner animation issues in Safari, causing
```
Comment (karlbaumhauer) [inline packages/components/src/docs/General/6-Known Browser Issues.mdx:16]:
```suggestion
The 'sd-spinner' component may experience spinner animation issues in Safari, causing them to appear wobbly.
```

File: packages/components/src/docs/General/6-Known Browser Issues.mdx [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+them to appear wobbly.
```
Comment (karlbaumhauer) [inline packages/components/src/docs/General/6-Known Browser Issues.mdx:17]:
```suggestion
```

**Review by karlbaumhauer — CHANGES_REQUESTED** [review]

**Comment by karlbaumhauer** [comment]
just a unnecessary line break. Other than that, LGTM.

**Review by karlbaumhauer — APPROVED** [review]

**Review by azraefendic — APPROVED** [review]

**Review by mariohamann — APPROVED** [review]

**Pull request merged**